code/__HELPERS/game.dm

```diff
@@ -405,11 +405,11 @@ datum/projectile_data
 	flick_overlay(I, viewing, duration)
 
 
-/proc/window_flash(client/C)
+/proc/window_flash(client/C, ignorepref = FALSE)
 	if(ismob(C))
 		var/mob/M = C
 		if(M.client)
 			C = M.client
```

Comment:
Why is it that given a client it gets the mob, then resets the original client to the mobs client?